### PR TITLE
New character suite: Symbols collection

### PR DESCRIPTION
**Pull request by srjfoo**
Character suite `symbols-collection`, contains astronomical, zodiac, apothecary and music symbols, grouped into two picker sets. The Zodiac symbols are combined with the VS15 variation selector to try
to force them to appear as normal type instead of as emoji, though this does not work in all cases.

Character suite can be viewed at [symbols-collection](https://www.pgdp.org/~srjfoo/c.branch/symbols-collection/tools/charsuites.php?charsuite=symbols-collection); used in a project at [Elegantní Tanecník - WordCheck Testing](https://www.pgdp.org/~srjfoo/c.branch/symbols-collection/project.php?id=projectID47127a26a1f9b&expected_state=P1.proj_avail).

### CODE REVIEW

File: SETUP/UNICODE.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ that are shown to the users in the proofreading interface character picker.
 Picker sets are optional and, if they are used, need not cover the entire set
 of codepoints in the character suite.
 
-The code currently ships with 4 character suites:
+The code currently ships with 9 character suites:
```
Comment (chrismiceli) [inline SETUP/UNICODE.md:45]: This will probably get out of date like before. Maybe reword to just ```The code currently ships with the following character suites:```?

File: pinc/charsuite-symbols-collection.inc [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+$charsuite = new CharSuite("symbols-collection", _("Symbols collection"));
+$charsuite->description = _("Astronomical, zodiac and a limited number of apothecary and music symbols. The zodiac symbols are provided with the variation selector, VS15, to try to force them to display as ordinary charcters instead of as emoji. This may not work for all fonts.");
```
Comment (chrismiceli) [inline pinc/charsuite-symbols-collection.inc:5]: what's our opinion on serial commas? :D

Comment (chrismiceli) [inline pinc/charsuite-symbols-collection.inc:5]: should we link to some wiki about variation selectors?

**Review by chrismiceli — APPROVED** [review]

File: SETUP/UNICODE.md (hunk 1 above)
Comment (cpeel) [reply SETUP/UNICODE.md:45]: This is a great idea.

File: pinc/charsuite-symbols-collection.inc (hunk 2 above)
Comment (cpeel) [reply pinc/charsuite-symbols-collection.inc:5]: Yes - Andrew, Sharon and I chatted about this and one is forthcoming.

File: SETUP/UNICODE.md (hunk 1 above)
Comment (srjfoo) [reply SETUP/UNICODE.md:45]: I agree. I'm waiting on a wiki page so I can update everything at the same time.

File: pinc/charsuite-symbols-collection.inc (hunk 2 above)
Comment (srjfoo) [reply pinc/charsuite-symbols-collection.inc:5]: That they're used inconsistently across the site. What would you suggest here? \<grin\>

Comment (chrismiceli) [reply pinc/charsuite-symbols-collection.inc:5]: I have no suggestion. I just usually use it so it stuck out to me. I approved because this shouldn't hold anything up. Just pointing it out in case there was some standard or you had a preference.

File: SETUP/UNICODE.md (hunk 1 above)
Comment (srjfoo) [reply SETUP/UNICODE.md]: Done.

File: pinc/charsuite-symbols-collection.inc (hunk 2 above)
Comment (srjfoo) [reply pinc/charsuite-symbols-collection.inc]: After reviewing the wikipedia page on serial commas and re-reading the blurb, I added it here (but neglected to add it in the `.md` file, which I've chosen to leave as-is unless there are mutterings on inconsistency :D).

Comment (srjfoo) [reply pinc/charsuite-symbols-collection.inc]: The wiki page has been started, but not completed as of this moment. However, I went ahead and added a link to it -- it will hopefully be ready by the time we need it.
